packages/twenty-docker/prod/twenty/inject-runtime-env.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+echo "Generating env-config.js file from runtime environment variables..."
+
+BASE_FILENAME="dist/front/env-config.js"
+rm -rf "./$BASE_FILENAME"
+
+{
+	echo "window._env_ = {"
+	echo "  REACT_APP_SERVER_BASE_URL: \"$REACT_APP_SERVER_BASE_URL\","
```

Comment:
If the frontend is served by the backend we could generate this file though the server application layer instead?
We'd only need this in the context of static/s3 deployment where front != back (so maybe twenty-infra specific?)